Disabled the update functionality so it doesn't break the package.
--- a/searchsploit
+++ b/searchsploit
@@ -33,7 +33,7 @@ function usage()
     echo "   -c            Perform case-sensitive searches; by default, searches will"
     echo "                 try to be greedy"
     echo "   -h, --help    Show help screen" 
-    echo "   -u            Update db from git" 
+#    echo "   -u            Update db from git" 
     echo "   -v            By setting verbose output, description lines are allowed to"
     echo "                 overflow their columns"
     echo
@@ -42,7 +42,7 @@ function usage()
     echo "======="
     echo " * Use any number of search terms you would like (minimum: 1)"
     echo " * Search terms are not case sensitive, and order is irrelevant"
-    echo " * When updating from git, searches will be ignored"
+#    echo " * When updating from git, searches will be ignored"
     exit 1
 }
 
@@ -71,14 +71,14 @@ for param in $@; do
 done
 
 # parse short arguments
-while getopts "chuv" arg $ARGS; do
+while getopts "chv" arg $ARGS; do
     if [ "$arg" = "?" ]; then
         usage >&2;
     fi
     case $arg in
         c) SCASE='';;
         h) usage >&2;;
-        u) UPDATE=1;;
+        u) UPDATE=0;;
         v) VERBOSE=1;;
     esac
     shift $((OPTIND-1))
